packages/serverpod/lib/src/server/serverpod.dart

```diff
@@ -729,6 +733,22 @@ class Serverpod {
     return _passwords[key];
   }
 
+  /// Registers passwords to be loaded from the env variables.
+  /// The password can be accessed with the [getPassword] method.
+  /// The envName is the name of the environment variable that
+  /// contains the password. The alias is the key used to access the
+  /// the password with the [getPassword] method.
+  /// The alias also maps to the name in the config/passwords.yaml file.
+  void loadCustomPasswords(
```

Comment:
Suggestion: Add information about the possible throw here as well.

Since this is the interface the user will actually be using it is important to have that information here.

Reply:
Isn't this actually the `merge` or `integrate` for the user?

Reply:
This is the public method indeed, my thinking here was that you are loading something from the env. (In the internal one we merge different maps basically, hence the difference). 

I don't think integrate is a good word here though. But +1 on the throw info.